src/Compilers/Core/Desktop/CommandLine/CommonCommandLineParser.cs

```diff
@@ -578,10 +578,52 @@ public static IEnumerable<string> SplitCommandLineIntoArguments(string commandLi
                 }))
             .Select(arg => arg.Trim())                                                                  // Trim whitespace
             .TakeWhile(arg => (!removeHashComments || !arg.StartsWith("#", StringComparison.Ordinal)))  // If removeHashComments is true, skip all arguments after one that starts with '#'
-            .Select(arg => CondenseDoubledBackslashes(arg).Unquote())                                   // Remove quotes and handle backslashes.
```

Comment:
What happened to CondenseDoubledBackslashes?

As far as I can tell this was the only place that referenced the method. If it is now unused it should be deleted.